src/coffee/sync/inventory-sync.coffee

```diff
@@ -20,7 +21,7 @@ InventoryUtils = require './utils/inventory'
 #     # do nothing
 class InventorySync extends BaseSync
 
-  @actionGroups = ['quantity', 'expectedDelivery']
+  @actionGroups = ['quantity', 'expectedDelivery', 'custom']
```

Comment:
It is defined [on line 22](https://github.com/sphereio/sphere-node-sdk/blob/master/src/coffee/sync/base-sync.coffee#L22) of the base-sync file.